osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/PowerQualityProfileResponseDataConverter.java

```diff
@@ -46,6 +46,11 @@
                 .PowerQualityProfileData();
     result.setLogicalName(this.mapperFacade.map(source.getLogicalName(), ObisCodeValues.class));
 
+    // add  source.getProfileTypes() to result
```

Comment:
This comment is not very useful I think. And it's outdate: profileTypes instead of profileType